test/unit/mem/buf/src/BufTest.cpp

```diff
@@ -81,10 +81,11 @@ static auto createVecFromIndexedFn()
 -> alpaka::Vec<TDim, TSize>
 {
     return
+        alpaka::
```

Comment:
IMO this is very hard to read if the code is spitted by to many `#ifdef`

Could we change it back to the old style and only add `vec::` to the secound `#ifdef` part
```C++
  #ifdef ALPAKA_CREATE_VEC_IN_CLASS
         alpaka::Vec<TDim, TSize>::template
  #else
         vec::alpaka::
  #endif
```